x-pack/legacy/plugins/epm/server/lib/agent/agent.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Mustache from 'mustache';
+
+interface Manifest {
+  vars: VarsEntry[];
+}
+
+interface VarsEntry {
+  name: string;
+  default: string;
+}
+/**
+ * This takes a manifest object as input and merges it with the input template.
+ * It returns the resolved template as a string.
+ */
+export function createInput(manifest: Manifest, inputTemplate: string): string {
+  const view: any = {};
```

Comment:
We should be able to avoid the `any` since we know these types are coming from `VarsEntry`. One option is
```suggestion
  const view: Record<VarsEntry["name"], VarsEntry["default"]> = {}
```